app/serializer.py

```diff
@@ -0,0 +1,30 @@
+import json
+from abc import ABC, abstractmethod
+from app.book import Book
+import xml.etree.ElementTree as ElemTree

```

Comment:
Order and group your methods according to PEP-8. Different groups of imports should be separated by blank line.